File: Python Code/Daily_Flash_Code_0/Week12/9Apr/69-DailyFlash_Solutions/09_April_Solution_One/Java/Program0.java
import java.io.*;
import java.util.*;

class ArrayOps{

	public static void main(String ... args)throws IOException{

		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

		System.out.println("Enter No of Elements : ");
		int m = Integer.parseInt(br.readLine());

		int arr1[] = new int[m];
		int arr2[] = new int[m];


	System.out.println("Enter Elements For Array 1");
	for(int i=0; i<m; i++){

		arr1[i] = Integer.parseInt(br.readLine());
	}

	System.out.println("Enter Elements For Array 2");
	for(int i=0; i<m; i++){

		arr2[i] = Integer.parseInt(br.readLine());
	}

	System.out.println("Subtraction is ");
	for(int i=0; i<m; i++){
			
		int sub = arr1[i] - arr2[i];
			if(sub < 0)	
				sub*=-1;
			System.out.printf("%d\t",sub);
			
	}		
	
	}
}

File: Python Code/Daily_Flash_Code_0/Week12/9Apr/69-DailyFlash_Solutions/09_April_Solution_One/CPP/Program0.cpp
#include<stdio.h>

int main(){

	int n = 0;

	printf("Enter Number of Elements : ");
	scanf("%d",&n);

	int arr1[n];
	int arr2[n];
	
	printf("Enter For array 1\n");
	for(int i=0; i<n; i++){

		scanf("%d",&arr1[i]);
	}

	printf("Enter For array 2\n");
	for(int i=0; i<n; i++){

		scanf("%d",&arr2[i]);
	}

	printf("Subtraction of Elements\n");
	for(int i=0; i<m; i++){
			
		int sub = arr1[i]-arr2[i];
			if( sub < 0)	
				sub*=-1;
			printf("%d\t",sub);
	}

	printf("\n");

	return 0;
}
